packages/p2p/browser-test/src/index.js

```diff
@@ -0,0 +1,224 @@
+import m from "mithril"
+import matchmaking from "../../clients/js"
+
+let connectionStatus = 'closed'
+let messages = []
+const room = 'squadChess'
```

Comment:
Let's all caps room here to `ROOM`. below when I saw the variable `room` I was confused about where it came from, but if it was formatted as a global config style variable that would have been clear.